.github/workflows/build.yml

```
@@ -0,0 +1,52 @@
+name: build
+
+on:
+  push:
+  workflow_dispatch:
+
+jobs:
+  linting:
+    name: run
+    runs-on: ubuntu-latest
+    env:
+        # set tools versions to latest supported by the CI runner,
+        # ubuntu 22.04 at the time of writing
```

This comment is not really helpful and can be removed:
```suggestion
        # set tools versions to latest supported by the CI runner,
        # ubuntu 22.04 at the time of writing
```